nixos/modules/virtualisation/qemu-guest-agent.nix

```diff
@@ -0,0 +1,39 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.qemuGuest;
+in {
+
+  options.services.qemuGuest = {
+      enable = mkOption {
+        type = types.bool;
+        default = false;
+        description = "Whether to enable the dspam spam filter.";
```

Comment:
Didn't you fix this before?

Reply:
right, I fucked up the merge of the changes from different branches sorry